### ISSUE TITLE
Eating all exceptions

### ISSUE THREAD

**Issue by calclavia** [NOVA-Team/NOVA-Wrapper-MC#53]
It's right now impossible to force Minecraft to crash with any exception during initialization phases, thanks to the eating of exceptions.

**Comment by Vexatos** [comment]
Maybe have some [Custom exception](https://github.com/MinecraftForge/MinecraftForge/blob/1.7.10/fml/src/main/java/cpw/mods/fml/client/CustomModLoadingErrorDisplayException.java) to force a crash?

**Comment by RX14** [comment]
print stacktrace then `System.exit()`

**Comment by Kubuxu** [comment]
Forge forbids `System.exit()` because drama (angry modders were doing `System.exit()` and users didn't know what was going on.)

Exceptions should be processed and passed over, possibly with some analysis of what cased them. On client we can use what @Vexatos shown to present the exception in nice way.

**Comment by RX14** [comment]
No, it reroutes it, it still exits, but after printing a message.

**Comment by RX14** [comment]
Custom exception however, is probably the best way to do this.

**Comment by calclavia** [comment]
Problem it seems is that our exception won't really exit. Forge tries to continue loading.

**Comment by Kubuxu** [comment]
Problem is also that a lot of exceptions are being just suppressed but still fail forge.

**Comment by RX14** [comment]
Can we please just stick a try catch print throw on every place where we hook up to a place where forge eats our exceptions?
